pkg/virtctl/imageupload/imageupload.go

```diff
@@ -723,3 +755,51 @@ func getUploadProxyURL(client cdiClientset.Interface) (string, error) {
 	}
 	return "", nil
 }
+
+// handleEventErrors checks PVC and DV-related events and, when encountered, returns appropiate errors
```

Comment:
Typo: `appropiate` should be `appropriate`

Reply:
Fixed :+1: